rest_service.go

```diff
@@ -79,6 +78,7 @@ func (s *Service) App() *gimlet.APIApp {
 	app.AddRoute("/process/{id}/metrics").Version(1).Get().Handler(s.processMetrics)
 	app.AddRoute("/process/{id}/signal/{signal}").Version(1).Patch().Handler(s.signalProcess)
 	app.AddRoute("/process/{id}/logs").Version(1).Get().Handler(s.getLogs)
+	app.AddRoute("/process/{id}/signal-trigger/{trigger-id}").Version(1).Patch().Handler(s.registerSignalTriggerID)
```

Comment:
"/process/{id}/signal/{triggerID}" 

i'm sort of against using hypens (I actually like them, but they're confusing in non-lisp languages,.)

Reply:
This is already being used by `signalProcess`, so I changed this to `/process/{id}/trigger/{trigger-id}`.
Should `/process/{id}/buildlogger-urls` also be changed?